src/contracts/interfaces/IRwaATokenManager.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IRwaATokenManager
+ * @author Aave
+ * @notice Defines the basic interface for the RWA aToken manager
+ */
+interface IRwaATokenManager {
+  /**
+   * @dev Grants authorized transfer role for an RWA aToken to an account
```

Comment:
we use @dev only for events, construtor and internal functions

Reply:
clear, thanks